repositories.json

```diff
@@ -40,8 +40,7 @@
   "packages": [{
     "name": "@salesforce/plugin-data",
     "type": "plugin"
-  }],
-  "private": true
+  }]
```

Comment:
would it be better to put it in as a library like the ones further down?
ex: https://github.com/salesforcecli/status/blob/7496d1d160d8ac5e12043169df9267e8d28ee0cf/repositories.json#L93

So it gets both version numbers posted?
![Screen Shot 2021-03-23 at 1 42 30 PM](https://user-images.githubusercontent.com/4261788/112200884-ad82dd00-8bdd-11eb-98bc-14770cc1e27c.png)